.github/workflows/IntegrationTest.yml

```diff
@@ -0,0 +1,55 @@
+name: IntegrationTest
+on:
+  push:
+    branches: [master]
+    tags: [v*]
+  pull_request:
+
+jobs:
+  test:
+    name: ${{ matrix.package.repo }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        julia-version: [1]
```

Comment:
Do we also want to test on LTS?

Reply:
we do not. In part because our dependencies might not support the LTS